pkg/config/pipeline.go

```diff
@@ -60,6 +61,9 @@ type PipelineConfig struct {
 	OnUpdate func(context.Context, *livekit.EgressInfo) `yaml:"-"`
 
 	Info *livekit.EgressInfo `yaml:"-"`
+
+	UploadsCounter      *prometheus.CounterVec   `yaml:"-"`
```

Comment:
can these be moved into the `stats` package? (probably with a new `HandlerMonitor` or similar)

Reply:
Thanks for the review and the suggestion, @frostbyte73 
I have refactored the code as you suggested.